src/app/backend/cert/manager.go

```diff
@@ -22,13 +22,13 @@ import (
 	certapi "github.com/kubernetes/dashboard/src/app/backend/cert/api"
 )
 
-// Implements cert/api/types.Manager interface. See Manager for more information.
+// Manager is used to implements cert/api/types.Manager interface. See Manager for more information.
```

Comment:
s/implements/implement

Reply:
Do you mean to remove the 's' form "implements" ? I fix it now.

Reply:
@floreks

Reply:
@maciaszczykm